### PR TITLE
[Polygon Edge] Validator hosting

### PR DESCRIPTION
**Pull request by ZeljkoBenovic**
This PR adds a Validator Hosting page with instructions/recommendations on how to properly host a validator node

### CODE REVIEW

**Review by lazartravica — APPROVED** [review]

File: docs/edge/validator-hosting.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+- Start `polygon-edge` service ( example: `sudo systemctl start polygon-edge.service` )
+- Finally, check the `polygon-edge` log output and make sure everything is running without any `[ERROR]` logs
+
+## Startup procedure
+
+Following is the desired flow of the startup procedure for Polygon Edge validator
+
+- Apply the latest OS patches on the validator node
+- Download the latest `polygon-edge` binary and place it in local instance `PATH`
+- Initialize one of the supported secrets manager using `polygon-edge secrets generate` CLI command
```
Comment (Aleksao998) [inline docs/edge/validator-hosting.md:224]: Maybe to link instructions from the cloud setup page. Also for other steps
ex Initialize one of the supported secrets manager using `polygon-edge secrets generate` CLI command ([read more](http://localhost:3000/docs/edge/get-started/set-up-ibft-on-the-cloud#step-1-initialize-data-folders-and-generate-validator-keys))

**Review by Aleksao998 — APPROVED** [review]
LGTM, really great article, thank you for writing this. 
I left some minor suggestions (most of them purely aesthetically)

File: docs/edge/validator-hosting.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+### Data storage
+
+The `data/` folder  containing the entire blockchain state should be mounted on a dedicated disk / volume allowing for
+automatic disk backups, volume extension and optionally mounting the disk/volume to another instance in case of failure.
+
+
+### Log files
+
+Log files need to be rotated on a daily basis (with a tool like `logrotate`).
+If configured without log rotation, log files could use up all the available disk space which could disrupt the validator uptime.
```
[the remark this answers is not in the document]
Comment (ZeljkoBenovic) [reply docs/edge/validator-hosting.md:85]: fixed f4322accff3c5d692cacefc4c260c5ed1cc4d917

File: docs/edge/validator-hosting.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+Below are the best practices for maintaining a running validator node of a Polygon Edge network.
+
+### Backup
+
+There are two types of backup procedures recommended for Polygon Edge nodes. 
+
+The suggestion is to use both, whenever possible, with the Polygon Edge backup being an always available option.
+
+### Volume backup
```
[the remark this answers is not in the document]
Comment (ZeljkoBenovic) [reply docs/edge/validator-hosting.md]: fixed in f4322accff3c5d692cacefc4c260c5ed1cc4d917

File: docs/edge/validator-hosting.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+        postrotate
+                /usr/bin/systemctl start polygon-edge.service
+        endscript
+}
+```
+
+
+Refer to the [Logging](#logging) section below for recommendations on log storage.
+
+### Backup
```
[the remark this answers is not in the document]
Comment (ZeljkoBenovic) [reply docs/edge/validator-hosting.md]: removed redundant Backup section f4322accff3c5d692cacefc4c260c5ed1cc4d917

File: docs/edge/validator-hosting.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+The logs outputted by the Polygon Edge nodes should:
+- be sent to an external data store with indexing and searching capabilities
+- have a log retention period of 30 days
+
+If this is your first time setting up a Polygon Edge validator, we recommend to start the node
+with the `--log-level=DEBUG` option to be able to quickly debug any issues you might face.
+
+The `--log-level=DEBUG` will make the node's log output be as verbose as possible.   
+Debug logs will drastically increase the size of the log file which must be taken into account when setting up
```
[the remark this answers is not in the document]
Comment (ZeljkoBenovic) [reply docs/edge/validator-hosting.md:145]: fixed in f4322accff3c5d692cacefc4c260c5ed1cc4d917

File: docs/edge/validator-hosting.md (hunk 1 above)
Comment (ZeljkoBenovic) [reply docs/edge/validator-hosting.md]: added links f4322accff3c5d692cacefc4c260c5ed1cc4d917

**Review by zivkovicmilos — APPROVED** [review]

**Pull request merged**